src/NServiceBus.AcceptanceTests/DeterministicGuid.cs

```diff
@@ -9,10 +9,10 @@ static class DeterministicGuid
         public static Guid Create(params object[] data)
         {
             // use MD5 hash to get a 16-byte hash of the string
-            using (var provider = new MD5CryptoServiceProvider())
+            using (var md5 = MD5.Create())
```

Comment:
FYI I did once a performance analysis between those two approaches and the static create approach was quite a bit slower. It is a long time ago since I ran it

https://github.com/danielmarbach/MicroBenchmarks/blob/master/MicroBenchmarks/Hashing/Hashing.cs

and I might be remembering incorrectly, or it doesn't even matter here.

Reply:
@danielmarbach the "CryptoServiceProvider" classes are all deprecated in .NET 6.